app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -12,17 +12,14 @@
 
 <%= render PageHeadingComponent.new.with_content(@presenter.heading) %>
 
-<p><%= @presenter.intro %></p>
-
-<%= render(AlertComponent.new) { t('two_factor_authentication.lockout_information') } %>
+<p class="maxw-mobile-lg margin-bottom-0"><%= @presenter.intro %></p>
 
 <%= validated_form_for @two_factor_options_form,
                        html: { autocomplete: 'off' },
                        method: :patch,
                        url: two_factor_options_path do |f| %>
   <div class="margin-bottom-4">
     <fieldset class="margin-0 padding-0 border-0">
-      <legend class="margin-bottom-2"><%= t('forms.two_factor_choice.legend') %>:</legend>
```

Comment:
1. Even if we don't want it visually, could we keep it for screen readers to have a description of the fieldset ([WCAG H71](https://www.w3.org/WAI/WCAG21/Techniques/html/H71))?
   - We can use `class="usa-sr-only"` for this, or `role="group" aria-label="..."` [like we've done elsewhere for similar](https://github.com/18F/identity-idp/blob/73e0c259d1dd6f9c590085e9d32c5e3815f520f8/app/views/two_factor_authentication/options/index.html.erb#L17) 
2. Otherwise or additionally, I see that we [ignore this key in `i18n-tasks.yml`](https://github.com/18F/identity-idp/blob/73e0c259d1dd6f9c590085e9d32c5e3815f520f8/config/i18n-tasks.yml#L102), but it's not obvious why we do that. I might think we could remove it there unless there's good reason for it.

Reply:
After searching, this seems like the only place it's used